test/ts-node-unit-tests/index.ts

```diff
@@ -0,0 +1,16 @@
+import { tap, spec } from 'node:test/reporters';
+import { run } from 'node:test';
+import { join } from 'node:path';
+
+import * as glob from 'glob';
+const files = glob.sync(join(__dirname, 'tests') + '/**/*.test.ts');
+
+// Workaround file for `node --test` not working with Windows and Node 20
+// TODO: consider removing when Node 22 is LTS
+
+run({
+    files,
+    watch: process.argv.includes('--watch')
+}).on('test:fail', () => {
```

Comment:
I don't get anything in the terminal on a fail (I changed a test to get a fail). Maybe something like `}).on('test:fail', (e) => {
    console.error('Test failed: ', e);` or equivalent could be added?

Reply:
🤔 hmm, the reporter gives me this on Windows (running in paralells) with Node 20: 
```sh
ℹ tests 61
ℹ suites 12
ℹ pass 60
ℹ fail 1
ℹ cancelled 0
ℹ skipped 0
ℹ todo 0
ℹ duration_ms 6669.1485

✖ failing tests:

test at test\ts-node-unit-tests\tests\global-utils.test.ts:2:1255
✖ correctly formats a date (6.924042ms)
  AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
  + actual - expected

  + 'Sunday,  1 Jan, 00:00:00'
  - 'Sunday,  2 Jan, 00:00:00'
              ^
      at TestContext.<anonymous> (C:\Users\goranslettemark\highcharts\test\ts-node-unit-tests\tests\global-utils.test.ts:72:13)
      at Test.runInAsyncScope (node:async_hooks:206:9)
      at Test.run (node:internal/test_runner/test:789:25)
      at Test.start (node:internal/test_runner/test:693:17)
      at node:internal/test_runner/test:1129:71
      at node:internal/per_context/primordials:487:82
      at new Promise (<anonymous>)
      at new SafePromise (node:internal/per_context/primordials:455:29)
      at node:internal/per_context/primordials:487:9
      at Array.map (<anonymous>) {
    generatedMessage: true,
    code: 'ERR_ASSERTION',
    actual: 'Sunday,  1 Jan, 00:00:00',
    expected: 'Sunday,  2 Jan, 00:00:00',
    operator: 'strictEqual'
  }
```

Reply:
What kind of failure did you add?

Reply:
Just some random typo in the expected array in modules.test.ts. I was using Node v18. After updating to v20.17.0 all is perfectly fine.